dts/bindings/i3c/nuvoton,npcx-i3c.yaml

```diff
@@ -50,3 +50,55 @@ properties:
     description: |
       Open Drain Frequency for the I3C controller. When undefined, use
       the controller default or as specified by the I3C specification.
+
+  instance-id:
+    type: int
+    description: |
+      Instance ID of the device, used to specify port number.
+      Bit[7:4] module id.
+      Bit[3:0] port id.
+
+  target-mode-en:
+    type: boolean
+    description: Initialized as controller or target mode.
+
+  secondary:
+    type: boolean
+    description: Initialized as a secondary master.
```

Comment:
I don't understand why you need both of these. It either boots as a controller (primary controller) or a target (secondary controller)

Reply:
The initial reason for doing this was to provide the option for the target device with secondary controller capability when it boots up as a target.

It means it can initialize as a pure target or as a target with a secondary controller.

![image](https://github.com/user-attachments/assets/3eee3dc4-4785-4091-b831-61fda2b58e70)

Reply:
Done.
Please refer https://github.com/alvsun/zephyr/blob/e1622037e214d7dd93a4f3246b2544e42ae4afa0/drivers/i3c/i3c_npcx.c#L2911 #